config/services.yml

```diff
@@ -412,6 +412,36 @@ https://cmr.earthdata.nasa.gov:
       - image: !Env ${TRAJECTORY_SUBSETTER_IMAGE}
         operations: ['variableSubset', 'spatialSubset', 'shapefileSubset']
 
+  - name: sds/HOSS-maskfill
+    # Provides polygon spatial subsetting for collections hosted in OPeNDAP
+    # Also provides variable and temporal subsetting through HOSS.
+    data_operation_version: '0.16.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/sds/HOSS
+    umm_s:
+      - S2300730272-XYZ_PROV
```

Comment:
verified same in prod as existing uat 
https://cmr.earthdata.nasa.gov/search/services.umm_json?concept_id=S2300730272-XYZ_PROV